temporal-serviceclient/src/main/java/io/temporal/serviceclient/RpcRetryOptions.java

```diff
@@ -93,6 +95,10 @@ public static final class Builder {
 
     private Duration maximumInterval;
 
+    // 0 is a valid value for maximumJitter, and yet, it is not the default value for maximumJitter.
+    // Use NaN instead as a marker for not configured jitter
+    private double maximumJitter = Double.NaN;
```

Comment:
I'm not a big fan of using NaN as a magic value for default. The main reason is that it may be tricky to pass / specify NaN through an external configuration system. 
I would advice using `-1` as a magic value for default here.